README.md

```diff
@@ -0,0 +1,61 @@
+# How to contribute a snippet
+
+1. Write the SQL to retrieve the data of interest.
+    * Try to put as much of the data wrangling logic into SQL as possible. This means there's less data wrangling code to write twice (e.g., once in [pandas](https://pandas.pydata.org/) for Python and again [dplyr](https://dplyr.tidyverse.org/) for R).
+    * Choose a good prefix for your `<my-query>.sql` filename. The file name helps users decide whether its useful to them. It will also become the name of the dataframe holding the query results (e.g. `<my-query>_df`).
+    * Put some comments in your query too. Look at the other `.sql` files for examples.
+    * Tip: Paste your SQL into the [BigQuery web UI](https://bigquery.cloud.google.com/) and click on the 'format' button to format it nicely.
```

Comment:
@deflaux given the impending changes to the workbench security boundary that will prevent users from using the BQ UI, should we propose alternatives that do not use the BQ UI?

Reply:
No need! The BigQuery UI can still be a great tool in our toolbox. Let's just provide world-readable OMOP tables with a few rows of fake data for testing purposes. Kinda like the fake data in the SQL unit tests.

See https://github.com/deflaux/code-snippets/blob/master/snippets/measurements_of_interest_summary_test.py#L50

Reply:
Sounds good!